account_operating_unit/views/account_move_view.xml

```diff
@@ -79,50 +64,32 @@
                     name="operating_unit_id"
                     options="{'no_create': True}"
                     groups="operating_unit.group_multi_operating_unit"
+                    readonly="state != 'draft'"
+                />
+                <field
+                    name="operating_unit_id"
+                    invisible="1"
+                    groups="!operating_unit.group_multi_operating_unit"
+                    readonly="state != 'draft'"
                 />
             </xpath>
             <field name="invoice_line_ids" position="attributes">
-                <attribute name="context">
-                    {'default_move_type': context.get('default_move_type'),
-                    'journal_id':
-                    journal_id, 'default_partner_id': commercial_partner_id,
-                    'default_currency_id': currency_id or company_currency_id,
-                    'operating_unit_id': operating_unit_id}
+                <attribute name="context" operation="update">
+                    {
+                        'operating_unit_id': operating_unit_id,
+                        'default_operating_unit_id': operating_unit_id
+                    }
                 </attribute>
             </field>
-            <xpath
-                expr="//field[@name='invoice_line_ids']/tree/field[@name='analytic_account_id']"
-                position="attributes"
-            >
-                <attribute name="domain">
-                    ['|', ('company_id', '=', False), ('company_id', '=',
-                    parent.company_id), '|', ('operating_unit_ids', '=',
-                    context.get('operating_unit_id', False)), ('operating_unit_ids',
-                    '=', False)]
-                </attribute>
-            </xpath>
-            <xpath
-                expr="//field[@name='invoice_line_ids']/form/sheet/group/field[@name='analytic_account_id']"
-                position="attributes"
-            >
-                <attribute name="domain">
-                    ['|', ('operating_unit_ids', '=', context.get('operating_unit_id',
-                    False)), ('operating_unit_ids', '=', False)]
-                </attribute>
-            </xpath>
             <field name="line_ids" position="attributes">
-                <attribute name="context">
+                <attribute name="context" operation="update">
```

Comment:
`operation="update"` will be only applied if you have a dependency on `base_view_inheritance_extension` I suppose, so probably you want to overwrite the original context like it was before

Reply:
Yes it's exactly that